packages/prisma-client/prisma/migrations/20221218211129_tree_text/schema.prisma

```diff
@@ -91,10 +90,4 @@ model Breakpoints {
   build   Build  @relation(fields: [buildId], references: [id])
   buildId String @id
   values  String @default("[]")
-}
-
-model DesignTokens {
```

Comment:
I think shouldn't be touched inside migrations

Reply:
ups